cli/src/cluster_query.rs

```diff
@@ -1827,6 +1843,71 @@ pub fn process_transaction_history(
     Ok(transactions_found)
 }
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct CliRentCalculation {
+    pub lamports_per_byte_year: u64,
+    pub lamports_per_epoch: u64,
+    pub rent_exempt_minimum_lamports: u64,
+}
+
+impl CliRentCalculation {
+    fn _write_str(
+        w: &mut dyn fmt::Write,
+        per_byte_year: &str,
+        per_epoch: &str,
+        exempt_minimum: &str,
+    ) -> fmt::Result {
+        writeln_name_value(w, "Rent per byte-year:", per_byte_year)?;
+        writeln_name_value(w, "Rent per epoch:", per_epoch)?;
+        writeln_name_value(w, "Rent-exempt minimum:", exempt_minimum)
+    }
+}
+
+impl fmt::Display for CliRentCalculation {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let per_byte_year = format!("{}", Sol(self.lamports_per_byte_year));
+        let per_epoch = format!("{}", Sol(self.lamports_per_epoch));
+        let exempt_minimum = format!("{}", Sol(self.rent_exempt_minimum_lamports));
```

Comment:
Can you use `build_balance_message()` so that the output is consistent with other cli commands?